src/web/app/components/question-responses/grouped-responses/grouped-responses.component.ts

```diff
@@ -60,4 +61,9 @@ export class GroupedResponsesComponent extends InstructorResponsesViewBase imple
     team.giver = `(${this.responses[0].allResponses[0].giverTeam})`;
     return team;
   }
+
+  toggleAddComment(commentTable: CommentTableModel): void {
+    commentTable.isAddingNewComment = !commentTable.isAddingNewComment;
```

Comment:
I think this shall be changed to
```suggestion
    triggerModelChangeForSingleResponse(responseId, Object.assign({ isAddingNewComment:  !commentTable.isAddingNewComment }, commentTable))
```